web/libs/editor/src/tags/object/Video/Video.js

```diff
@@ -88,6 +90,8 @@ const TagAttrs = types.model({
   height: types.optional(types.string, "600"),
   timelineheight: types.maybeNull(types.string),
   muted: false,
+  defaultplaybackspeed: types.optional(types.number, 1),
+  minplaybackspeed: types.optional(types.number, 1),
```

Comment:
These params are parsed from XML, they can't be just numbers unfortunately. But we can use `union` and convert them to numbers in `afterCreate()`.
```suggestion
  defaultplaybackspeed: types.optional(types.union(types.string, types.number), "1"),
  minplaybackspeed: types.optional(types.union(types.string, types.number), "0.25"),
```